core/src/saros/account/XMPPAccountStore.java

```diff
@@ -264,6 +227,15 @@ private synchronized void saveAccounts() {
     LOG.debug("saved " + accounts.size() + " account(s)");
   }
 
+  private XStream createXStream() {
+    XStream xStream = new XStream();
+    xStream.alias("accounts", ImmutablePair.class);
+    xStream.aliasField("activeAccount", ImmutablePair.class, "left");
```

Comment:
Isn't there another way to retrieve the field name left & right in a compile safe fashion ?

Reply:
It seems as if xstream does not support a compile safe way.